styles.css

```diff
@@ -1,4 +1,92 @@
 * {
   margin: 0;
-  padding: 0;
+  background-color: #1f1e1b;
+  font-family: 'Inconsolata', monospace;
+}
+
+header {
+  display: flex;
+  margin: 1em;
+  justify-content: space-between;
+}
+
+img {
+  display: flex;
+  height: 2.5em;
+}
+
+.menu-button {
+  display: inline-block;
+  cursor: pointer;
+  align-self: center;
+}
+
+.menu-button-bar1,
+.menu-button-bar2,
+.menu-button-bar3 {
+  width: 36px;
+  height: 5px;
+  background-color: #f0e3ca;
+  margin: 6px 0;
+  transition: 0.4s;
+}
+
+/* Rotate first bar with JavaScript - JS code pending */
+.change .menu-button-bar1 {
+  transform: translate(0, 11px) rotate(-45deg);
+}
+
+/* Fade out the second bar with JavaScript - JS code pending */
+.change .menu-button-bar2 { opacity: 0; }
+
+/* Rotate last bar with JavaScript - JS code pending */
+.change .menu-button-bar3 {
+  transform: translate(0, -11px) rotate(45deg);
+}
```

Comment:
Hi! @GerardoGomezG Great job! This section is a good idea, Can you share where you got it from? 😄

Reply:
Hello @danyhoshi !

You'll find the requested info here: [Hamburger button](https://blog.hubspot.com/website/hamburger-button)

Thanks